File: remove_shadows.js
const fs = require('fs');
const path = require('path');

const directory = "C:\\Users\\Vedansh\\Desktop\\Kriyeta Interview Website\\src\\components";

// Regex patterns to catch tailwind shadow utilities
const patterns = [
    /shadow-\[.*?\]/g,
    /\bshadow-(sm|md|lg|xl|2xl|inner)\b/g,
    /\bdrop-shadow(-\w+)?(\[.*?\])?\b/g,
    /ring-1 ring-white\/\d+/g // also convert old white rings
];

fs.readdirSync(directory).forEach(filename => {
    if (filename.endsWith(".jsx")) {
        const filepath = path.join(directory, filename);
        let content = fs.readFileSync(filepath, 'utf-8');
        const originalContent = content;
        
        // Remove shadows
        patterns.forEach(pattern => {
            content = content.replace(pattern, '');
        });
        
        // Cleanup double spaces left by removal
        content = content.replace(/  +/g, ' ');
        content = content.replace(/ className=" "/g, ' className=""');
        
        if (content !== originalContent) {
            fs.writeFileSync(filepath, content, 'utf-8');
            console.log(`Removed heavy shadows from ${filename}`);
        }
    }
});

console.log("Shadow stripping complete.");

File: remove_shadows.py
import os
import re

directory = r"C:\Users\Vedansh\Desktop\Kriyeta Interview Website\src\components"

# Regex patterns to catch tailwind shadow utilities
patterns = [
    r'shadow-\[.*?\]',
    r'\bshadow-(sm|md|lg|xl|2xl|inner)\b',
    r'\bdrop-shadow(-\w+)?(\[.*?\])?\b',
    r'ring-1 ring-white/\d+' # also convert old white rings to clean borders
]

for filename in os.listdir(directory):
    if filename.endswith(".jsx"):
        filepath = os.path.join(directory, filename)
        with open(filepath, 'r', encoding='utf-8') as f:
            content = f.read()
        
        original_content = content
        
        # Remove shadows
        for pattern in patterns:
            content = re.compile(pattern).sub('', content)
        
        # Cleanup double spaces left by removal
        content = re.sub(r'  +', ' ', content)
        
        if content != original_content:
            with open(filepath, 'w', encoding='utf-8') as f:
                f.write(content)
            print(f"Removed heavy shadows from {filename}")

print("Shadow stripping complete.")
